projects/admin/src/app/routes/corporate-bodies-route.ts

```diff
@@ -48,13 +48,11 @@ export class CorporateBodiesRoute extends BaseRoute implements RouteInterface {
             label: 'corporate-bodies',
             component: ContributionBriefComponent,
             detailComponent: ContributionDetailViewComponent,
+            searchFilters: [
+              this.advancedSearchFilter()
+            ],
             aggregationsOrder: ['sources'],
-            aggregationsExpand: ['sources'],
-            // use simple query for UI search
-            preFilters: {
-              type: 'bf:Organisation',
```

Comment:
We lost this filter ?

Reply:
This is replaced by the `searchFilters`.